.travis.yml

```diff
@@ -1,14 +1,23 @@
 sudo: true
 
 dist: xenial
-
 language: java
 
 jdk:
   - openjdk8
 
+services:
+  - docker
+before_install:
+  - docker pull openjdk:8-jdk-alpine
```

Comment:
I don't understand why you are moving the compilation/test to the an internal docker machine. Technically the machine where the compilation/testing is happening is already a clean machine with ojdk8 so there should not be any difference.
In fact I thought that it would be better to start the postgrest server that is required for the tests instead. What are your thoughts?

Reply:
I understand this is strange because this is Java and it should work in a similar way on the same jdk on default settings, but there is a problem to test code in default Travis environment. There is a problem with Jackson-core dependency, Travis can't find this dependency and throw an exception(NoClassDefFoundError) on JsonParser.Feature enum. I don't know why it happens, I  spent a lot of time on that problem and the only solution that I found is using docker container with preinstalled OpenJDK.